src/sinks/datadog_metrics.rs

```diff
@@ -225,6 +237,41 @@ fn encode_namespace(namespace: &str, name: &str) -> String {
     }
 }
 
+fn stats(values: &Vec<f64>, counts: &Vec<u32>) -> Option<DatadogStats> {
+    let mut samples = Vec::new();
+    for (v, c) in values.iter().zip(counts.iter()) {
+        for _ in 0..*c {
+            samples.push(*v);
+        }
+    }
+
+    if samples.is_empty() {
+        return None;
+    }
+
+    samples.sort_by(|a, b| a.partial_cmp(b).unwrap());
+
+    let length = samples.len() as f64;
+    let min = samples.first().unwrap();
+    let max = samples.last().unwrap();
```

Comment:
Same with these unwraps. Should we check at the beginning of the function that `values.len() >= 2 && values.len() == counts.len()` and return `None` otherwise?

Reply:
Right, thanks! There was actually a bug there. Should be ok now.